public/wp-content/themes/ColoredCow/mailer.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * Template Name: Mailer
+ */
+
+require_once get_template_directory() . '/vendor/mandrill/mandrill/src/Mandrill.php';
+function register_email($guest_email,$guest_name,$host_email,$host_name)
```

Comment:
We can use a Wrapper class for Mandrill. I see a lot of code repetition in both functions. Can you read about creating a wrapper class and class's constructor?